src/test/java/com/amazon/opendistroforelasticsearch/knn/KNNRestTestCase.java

```diff
@@ -413,4 +420,16 @@ protected String getIndexSettingByName(String indexName, String settingName) thr
                         .get("settings");
         return (String)settings.get(settingName);
     }
+
+    /**
+     * Clear cache
+     *
+     * This function is a temporary workaround. Right now, we do not have a way of clearing the cache except by deleting
+     * an index or changing k-NN settings. That being said, this function bounces a random k-NN setting in order to
+     * clear the cache.
+     */
```

Comment:
Thanks for the comment here. Please create an issue so that it would be easy to track and get back

Reply:
#68 exists for the flush API. Should I create another for this? Or add comment to that issue?

Reply:
Adding a comment to the existing issue should be fine.